### PR TITLE
sortmerna: add linux-aarch64 build and update to 4.3.7

### PR DESCRIPTION
**Pull request by martin-g**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by martin-g** [comment]
Fix for Linux ARM64 upstream - https://github.com/sortmerna/sortmerna/pull/417

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
```
mgrigorov in 🌐 euler-arm-22 in /tmp/sortmerna took 7s 
❯ tar xvf sortmerna-4.3.7-h7021222_0.tar.bz2 
info/files
info/run_exports.json
info/test/run_test.sh
info/recipe/build.sh
info/hash_input.json
info/paths.json
info/index.json
info/recipe/meta.yaml.template
info/recipe/meta.yaml
info/recipe/conda_build_config.yaml
info/licenses/LICENSE.LESSER.txt
info/about.json
info/git
bin/sortmerna

mgrigorov in 🌐 euler-arm-22 in /tmp/sortmerna 
❯ file ./bin/sortmerna 
./bin/sortmerna: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped

mgrigorov in 🌐 euler-arm-22 in /tmp/sortmerna 
❯ ./bin/sortmerna

[process:1388] ERROR: Missing required command options

  Program:      SortMeRNA version 4.3.7
  Copyright:    2016-2020 Clarity Genomics BVBA:
                Turnhoutseweg 30, 2340 Beerse, Belgium
                2014-2016 Knight Lab:
                Department of Pediatrics, UCSD, La Jolla
                2012-2014 Bonsai Bioinformatics Research Group:
                LIFL, University Lille 1, CNRS UMR 8022, INRIA Nord-Europe
  Disclaimer:   SortMeRNA comes with ABSOLUTELY NO WARRANTY; without even the
                implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
                See the GNU Lesser General Public License for more details.
  Contributors: Jenya Kopylova   jenya.kopylov@gmail.com
                Laurent Noé      laurent.noe@lifl.fr
                Pierre Pericard  pierre.pericard@lifl.fr
                Daniel McDonald  wasade@gmail.com
                Mikaël Salson    mikael.salson@lifl.fr
                Hélène Touzet    helene.touzet@lifl.fr
                Rob Knight       robknight@ucsd.edu
```

LGTM!

**Comment by martin-g** [comment]
@BiocondaBot please add label

File: recipes/sortmerna/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+{% set version = '4.3.7' %}
 
 package:
-  name: sortmerna
+  name: {{ name }}
   version: {{ version }}
-  
+
 # TODO:
 #   - for now using pre-built binaries due complexities with C++17 and the dependencies.
```
Comment (martin-g) [inline recipes/sortmerna/meta.yaml:9]:
```suggestion
```

File: recipes/sortmerna/meta.yaml [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
+{% set name = 'sortmerna' %}
+{% set version = '4.3.7' %}
 
 package:
-  name: sortmerna
+  name: {{ name }}
   version: {{ version }}
-  
+
 # TODO:
```
Comment (martin-g) [inline recipes/sortmerna/meta.yaml:8]:
```suggestion
```

File: recipes/sortmerna/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -x
```
Comment (martin-g) [inline recipes/sortmerna/build.sh:3]:
```suggestion
set -xe
```

**Review by mencian — APPROVED** [review]
LGTM, thanks @martin-g!

**Pull request merged**